staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/testcase.go

```diff
@@ -408,6 +408,11 @@ func NewNonMutatingTestCases(url *url.URL) []ValidatingTest {
 
 			SkipBenchmark: true,
 			ExpectAllow:   true,
+			ExpectAnnotations: map[string]string{
+				"validating.webhook.admission.k8s.io/failed-open": "internalErr A",
+				"validating.webhook.admission.k8s.io/failed-open": "internalErr B",
+				"validating.webhook.admission.k8s.io/failed-open": "internalErr C",
```

Comment:
this unit test demonstrates the problem with this approach... a map can't have multiple entries for the same key